test/tests/executioners/nl_divergence_tolerance/nl_divergence_tolerance.i

```diff
@@ -50,8 +50,8 @@
   nl_max_its = 20
   nl_div_tol = 1.0e+4
 
```

Comment:
@frombs `make nl_div_tol = 1.0e+4 small ` will have the test more robust. Change to `nl_div_tol=10`

Here we just want to make sure nobody delete `nl_div_tol`